gramjs/client/updates.ts

```diff
@@ -128,7 +128,7 @@ export async function _dispatchUpdate(
         let event = args.update;
         if (event) {
             if (!client._selfInputPeer) {
-                client.getMe(true).catch(() => {
+                await client.getMe(true).catch(() => {
```

Comment:
Without this `await`, `builder.build()` in line 139 of `updates.ts` is called before `client._selfInputPeer` has been set, so `undefined` is passed as `selfId`. This then causes a `new Api.PeerUser` to be constructed in line 150 of `NewMessage.ts` with its `userId` set to `undefined`.

Reply:
this was kinda done on purpose because of QR logins a while ago. Can you switch this to a try catch instead of await .catch()  ?

I'll test it a bit and if everything is fine I'll merge

Reply:
Replaced the `.catch()` with `try/catch`.